tools/wptserve/wptserve/handlers.py

```diff
@@ -298,6 +302,12 @@ def _load_file(self, request, response, func):
         """
         path = filesystem_path(self.base_path, request, self.url_base)
 
+        # Find a default Python file if the specified path is a directory
+        if os.path.isdir(path):
+            default_py = next(pathlib.Path(path).glob("default*.py"), None)
```

Comment:
Good point. I originally wrote this to assume it should just look for default.py, and only after that thought that test authors might need substitution or other pipe support. I had considered making the case where wptserve found multiple "default*.py" files an error instead of silently selecting one, but considered that there might be a reason why someone's handler might need to happen to match default*.py. I think having some deterministic behavior makes sense, and I've gone with lexicographical order as the most obvious choice. Thanks!